cmd/heimdalld/cmd/commands.go

```diff
@@ -482,6 +482,10 @@ func generateValidatorKey() *cobra.Command {
 // importValidatorKey imports validator private key from the given file path
 func importValidatorKey() *cobra.Command {
 	cdc := codec.NewLegacyAmino()
+	cdc.RegisterInterface((*crypto.PubKey)(nil), nil)
+	cdc.RegisterInterface((*crypto.PrivKey)(nil), nil)
+	cdc.RegisterConcrete(secp256k1.PubKey{}, "tendermint/PubKeySecp256k1", nil)
```

Comment:
Should be `cometbft/PubKeySecp256k1eth`

Reply:
I believe we still use the `tendermint` namespace no ? I think the plan is to migrate to the new naming when we switch to upstream comet.

Reply:
Okay seems I missed the detail where we asked everyone to also migrate the name. Updated [here](https://github.com/0xPolygon/heimdall-v2/pull/399/commits/a66b224eaa4ae07bf73789537434885829a1983b)